server/utils/utils.ts

```diff
@@ -184,3 +184,26 @@ export const getWeekOfDatesStartingMonday = (
 
   return { weekOfDates, previousWeek, nextWeek }
 }
+
+export const longPrisonerDateTimePretty = (dateToFormat: string): string => {
```

Comment:
As discussed, fine to leave these for now if they're working OK but we can probably simplify as a future change because we have other functions / Nunjucks filters that do same / similar.